checkbox-provider-ce-oem/bin/cpufreq_governors.py

```diff
@@ -792,27 +778,21 @@ def main():
     if args.debug:
         logger.setLevel(logging.DEBUG)
 
-    info = CPUScalingInfo()
+    handler = CPUScalingHandler()
     if args.policy_resource:
-        info.print_policies_list()
-        return 0
+        handler.print_policies_list()
+        sys.exit(0)
 
     test = CPUScalingTest(policy=args.policy)
     if args.driver_detect:
-        return 0 if test.test_driver_detect() else 1
+        sys.exit(0) if test.test_driver_detect() else sys.exit(1)
 
-    exit_code = 0
     try:
         test.print_policy_info()
+        if args.governor not in handler.governors:
+            probe_governor_module(args.governor)
         if not getattr(test, "test_{}".format(args.governor))():
-            exit_code = 1
+            sys.exit(1)
     except AttributeError:
         logging.exception("Given governor is not supported")
-        return 1
-
-    test.restore_governor()
-    return exit_code
-
-
-if __name__ == "__main__":
```

Comment:
The policy from Cert Team

- The if __name__ == "__main__": is abused, should only call main

Reference: https://github.com/canonical/checkbox/pull/586#pullrequestreview-1801698924